x/liquidity/spec/01_concepts.md

```diff
@@ -1,60 +1,83 @@
-<!--
-order: 1
--->
+<!-- order: 1 -->
 
-# Concepts
+ # Concepts
 
-## The Liquidity module on the Cosmos-SDK
+## Liquidity Module
 
-The liquidity module serves AMM style decentralized exchange on the Cosmos-SDK. AMM style exchange provides unique coin swap model for its users, liquidity providers and swap requestors.
+The Liquidity Module lets you implement a decentralized token exchange (DEX) on any Cosmos SDK-based network. Any user can create a liquidity pool with a pair of tokens, provide liquidity by depositing reserve tokens into the liquidity pool, and trade tokens using the liquidity pool.
 
-### Democratized Liquidity Providing
+### How the Liquidity Module works
 
-AMM allows liquidity providers to play market maker roles without technically sophisticated real-time orderbook management and significant capital requirement. The liquidity provides only need to deposit coins into liquidity pools, and monitor asset composition changes and accumulated fee rewards from liquidity providing.
+These features of the Liquidity Module create incentives to transfer tokens:
 
-It results in democratized liquidity providing activities, hence lowering the cost of liquidity and more enriched quality liquidity provided on the AMM exchange.
```

Comment:
@dongsam in the original doc, what do we mean with "democratized liquidity providing activities"?
what is providing activities? are we talking about depositing tokens and trading tokens?